### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by pabs3**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -26434,6 +26488,8 @@ unrelying->underlying
 unrepentent->unrepentant
 unrepetant->unrepentant
 unrepetent->unrepentant
+unreplacable->non-replaceable
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:26491]: This appears to be a real word: https://www.lexico.com/definition/unreplaceable

Irreplaceable would be another suggestion too anyway.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -26434,6 +26488,8 @@ unrelying->underlying
...
+unreplacalbe->non-replaceable
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:26492]: Again

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -25028,22 +25052,52 @@ tempory->temporary
...
 tempreatures->temperatures
+temprement->temperament
+tempremental->temperamental
+temproaily->temporarily
+temproal->temporal
 temproarily->temporarily
+temproarly->temporarily
 temproary->temporary
+temproay->temporary
+temproily->temporarily
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:25087]: Suggest temporally too

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -25028,22 +25052,52 @@ tempory->temporary
...
+tempraarly->temporarily
+tempraary->temporary
+tempraay->temporary
+tempraily->temporarily
+tempral->temporal
+temprament->temperament
+tempramental->temperamental
+tempraraily->temporarily
+tempraral->temporal
+tempraraly->temporarily
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:25067]: Suggest temporally too

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -21671,13 +21674,34 @@ repetions->repetitions
...
+replacability->replaceability
 replacable->replicable, replaceable,
+replacables->replaceables
+replacacing->replacing
+replacaiblity->replaceability
+replacalbe->replaceable
+replacalbes->replaceables
+replacament->replacement
+replacaments->replacements
+replacas->replicas
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:21687]: Suggest replaces too

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -21671,13 +21674,34 @@ repetions->repetitions
 repetive->repetitive
 repid->rapid
 repitition->repetition
+replaca->replica
+replacability->replaceability
 replacable->replicable, replaceable,
+replacables->replaceables
+replacacing->replacing
+replacaiblity->replaceability
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:21682]: Suggest replicability too

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -21671,13 +21674,34 @@ repetions->repetitions
 repetive->repetitive
 repid->rapid
 repitition->repetition
+replaca->replica
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:21677]: Suggest replace too

**Comment by pabs3** [comment]
Fixed all the issues you mentioned, thanks.

**Pull request merged**